tests/kafkatest/benchmarks/streams/streams_simple_benchmark_test.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, test_context):
 
 
     @cluster(num_nodes=9)
-    @matrix(test=["all"], scale=[1])
+    @matrix(test=["produce", "consume", "count", "processstream", "processstreamwithsink", "processstreamwithstatestore", "processstreamwithcachedstatestore", "kstreamktablejoin", "kstreamkstreamjoin", "ktablektablejoin"], scale=[1, 2, 3])
```

Comment:
"all" was meant to be a convenience function when running locally, e.g.,  in a debugger. Not at scale. The reason is that a bit subtle. If I want to see how something like `count` scales, I don't want to mix the `count` run on one instance, with a random function that could still be running on another, e.g., `consume`. So all instances must be running the same function for the scale test to make sense.